functional_test/pet_store_client.py

```diff
@@ -133,3 +133,9 @@ def signup_user(self, user):
         return self.session.request('POST', url, self.headers, json.dumps(user))
 
 
+    def find_user_by_name(self, userName):
+        """
+        Get user by userName
+        """
+        url = urljoin(self.index_url, "/users/{0}".format(userName))
+        return self.session.request('GET', url, self.headers)
```

Comment:
add a newline

Reply:
done

Reply:
Just realised that you propably ment to add new line after assert and return. Is that right ?